src/background/controllers/browser-utils.ts

```diff
@@ -33,7 +33,7 @@ class BrowserUtils {
 
   openPopup = async () => {
     if (this.cached) {
-      this.focusWindow(this.cached.id);
+      await this.focusWindow(this.cached.id);
```

Comment:
Shouldn't we await for the `focusWindow()`?

Reply:
Yes, I have await here.